tests/test_data/demo.sql

```diff
@@ -1798,6 +1801,7 @@ INSERT INTO public.action_type_translations VALUES (6, -98, -97, 'Simulation', '
 INSERT INTO public.action_types VALUES (-99, false, true, true, true, true, true, true, true, true, true, false);
 INSERT INTO public.action_types VALUES (-98, false, true, true, false, true, true, true, true, true, true, false);
 INSERT INTO public.action_types VALUES (-97, false, true, true, false, true, true, true, true, true, true, false);
+INSERT INTO public.action_types VALUES (-96, false, true, true, false, true, true, true, true, true, true, false);
```

Comment:
```suggestion
```
(3/3) This sql dump represents an older state of SampleDB and is used to test the migrations.
As the Template action type is added in a migration it should not be created here.
This leads to a failing migration test.
The name `demo.sql` might be a little misleading here..